src/scrabble.js

```diff
@@ -1,6 +1,55 @@
+const POINTS = {
+  A: 1, E: 1, I: 1, O: 1, U: 1, L: 1, N: 1, R: 1, S: 1, T: 1,
+  D: 2, G: 2, B: 3, C: 3, M: 3, P: 3, F: 4, H: 4, V: 4, W: 4, Y: 4, K: 5,
+  J: 8, X: 8, Q: 10, Z: 10, '}': 0, '{': 0, '[': 0, ']': 0
+};
+
 class Scrabble {
-  // Write your implementation here
+  constructor(string) {
+    this.string = string
+  }
+  score() {
+    let sum = 0
```

Comment:
This is being really nit-picky, but you have a really good solution so I thought I'd mention it. It would be more optimal to have your `let sum = 0` to be underneath your `if (this.string === null)`. That way, the computer doesn't need to create a variable that will never be used (if the string is null)

Reply:
that's true the function will die after it returns something, but I thought it would not affect the function's performance or the memory as the integer takes only 4 bytes, but even though thanks for pointing that.